src/ddmd/func.d

```diff
@@ -3189,6 +3182,30 @@ extern (C++) final class UnitTestDeclaration : FuncDeclaration
         return FuncDeclaration.syntaxCopy(utd);
     }
 
+    final void setIdentifier()
+    {
+        ident = createIdentifier(loc, _scope);
```

Comment:
Does this replace the temporary one above?

Reply:
It does. I added ddoc to explain that.